src/neo/Network/P2P/Payloads/Witness.cs

```diff
@@ -6,24 +6,33 @@
 
 namespace Neo.Network.P2P.Payloads
 {
+    /// <summary>
+    /// Represents a witness of an <see cref="IVerifiable"/> object.
+    /// </summary>
     public class Witness : ISerializable
     {
-        /// <summary>
-        /// This is designed to allow a MultiSig 21/11 (committee)
-        /// Invocation = 11 * (64 + 2) = 726
-        /// </summary>
+        // This is designed to allow a MultiSig 21/11 (committee)
+        // Invocation = 11 * (64 + 2) = 726
         private const int MaxInvocationScript = 1024;
 
-        /// <summary>
-        /// Verification = m + (PUSH_PubKey * 21) + length + null + syscall = 1 + ((2 + 33) * 21) + 2 + 1 + 5 = 744
-        /// </summary>
+        // Verification = m + (PUSH_PubKey * 21) + length + null + syscall = 1 + ((2 + 33) * 21) + 2 + 1 + 5 = 744
```

Comment:
summary?

Reply:
They are `private` fields, and no one can see the XML.